Sources/SwiftGenKit/Parsers/Fonts/FontsParser.swift

```diff
@@ -35,18 +35,41 @@ public enum Fonts {
 
 // MARK: - Helpers
 
+private extension Data {
+    var isTrueTypeFont: Bool {
+        var values = [UInt8](repeating: 0, count:5)
+        self.copyBytes(to: &values, count: 5)
+        let firstBytesOfTTF: [UInt8] = [0x00, 0x01, 0x00, 0x00, 0x00] // https://en.wikipedia.org/wiki/List_of_file_signatures
```

Comment:
These could be constants, like `trueTypeFontHeaderBytes` and such. And then a simple function on `Path` to check a file type given some header bytes.

Also avoids having a fixed "count" in the line above, as you can get the `.count` from the constant.